src/TEC/Plugin.php

```diff
@@ -112,6 +112,9 @@ public function register(): void {
 			return;
 		}
 
+		// Run the label hooks
+		add_action( 'plugins_loaded', [ $this, 'init_label_hooks' ] );
```

Comment:
We won't want to run our tickets/commerce/attendee etc hooks, out of the context of the All Import plugin.

In other words, as this is setup now it will run these hooks on every page load when the `plugins_loaded` hook runs. As long as the extension is active, it could negatively affect other site functionality.

We want to constraint the add-on hooks to only fire when needed. We could target the All Import pages these are relevant for, which looks like they need to be broken out into a "these hooks fire on page A" function, and a "these hooks fire on import" function. 

Does that make sense?

Reply:
Looks like you started to do the separation, but the targeting logic is missing.

Reply:
I added some logic, did the best I could. :)

Reply:
I think that'll do it. Thanks!